### PR TITLE
[map] fix webdriver tests

### PR DESCRIPTION
**Pull request by chejennifer**
- fix broken map webdriver test
- add shared function that should be able to help with flaky tests

### CODE REVIEW

File: server/webdriver_tests/shared.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+LOADING_WAIT_TIME = 3
+"""Common library for functions used by multiple webdriver tests"""
+
+
+def wait_for_loading(driver):
+    screen_present = EC.visibility_of_element_located((By.ID, 'screen'))
+    screen_hidden = EC.invisibility_of_element_located((By.ID, 'screen'))
+    while (True):
+        try:
```
Comment (shifucun) [inline server/webdriver_tests/shared.py:27]: do not quite understand the logic here? Can you add some comment?

**Review by shifucun — APPROVED** [review]
Thanks for the fix, super helpful!

File: server/webdriver_tests/shared.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as EC
+
+LOADING_WAIT_TIME = 3
```
Comment (beets) [inline server/webdriver_tests/shared.py:19]: it's usually good practice to add the unit to these time based constants (e.g. loading_wait_time_ms)

File: server/webdriver_tests/shared.py (hunk 1 above)
Comment (beets) [reply server/webdriver_tests/shared.py:27]: should we also add some max retries?

**Review by beets — APPROVED** [review]
thanks for the fix!

File: server/webdriver_tests/shared.py (hunk 1 above)
Comment (chejennifer) [reply server/webdriver_tests/shared.py:35]: added comments and max retries

File: server/webdriver_tests/shared.py (hunk 2 above)
Comment (chejennifer) [reply server/webdriver_tests/shared.py]: done

**Pull request merged**